frontend/src/components/Contexts/MissionListsContext.tsx

```diff
@@ -142,13 +175,15 @@ export const useMissions = (): MissionsResult => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [BackendAPICaller.accessToken])
```

Comment:
Currently missionDefinitions don't show initially and on reload. Adding equivalent useEffect with API-call like this above solves that